Makefile

```diff
@@ -1,9 +1,18 @@
-TEMPLATE_DIR = $(HOME)/Library/Developer/Xcode/Templates/MVVM-C\ Templates
 
-install: $(TEMPLATE_DIR)
-	cp -R *.xctemplate $(TEMPLATE_DIR)
+TEMPLATE_NAMES = 'MVVM-C' 'CellKit'
```

Comment:
Nešlo by to nějak zjednodušit a vzít všechny `*/*.xctemplate` soubory a zkopírovat je?